packages/integrations/tailwind/src/index.ts

```diff
@@ -66,6 +66,20 @@ async function getUserConfig(root: URL, configPath?: string, isRestart = false)
 	}
 }
 
+function getViteConfiguration(isBuild: boolean, tailwindConfig: TailwindConfig) {
+	const postcssPlugins = [tailwindPlugin(tailwindConfig)];
+	if (isBuild) {
+		postcssPlugins.push(autoprefixerPlugin());
+	}
+	return {
+		css: {
+			postcss: {
+				plugins: postcssPlugins,
```

Comment:
@bluwy Setting an inline PostCSS config in Vite, has the effect that Vite no longer looks for `postcss.config.cjs`. 

See: 
> Note if an inline config is provided, Vite will not search for other PostCSS config sources.
> https://vitejs.dev/config/shared-options.html#css-postcss 

That results in by adding the Tailwind Integration 3.0.0-beta.x, `postcss.config.cjs` no longer work as described in the Astro Docs.  
https://docs.astro.build/en/guides/styling/#postcss

Reply:
Hmm that's true, but I think that happens before this PR too as Astro was assigning to the property internally:

https://github.com/withastro/astro/pull/5717/files#diff-025f8338e5be413b0f65c74bc50d6484c6bc9bb4f78847da0ba896b8e9569390L153-L155

I guess for the tailwind integration we need to read `postcss.config.js` too to resolve this.

Reply:
Confirming, trying latest `@beta` packages makes Tailwind to ignore `postcss.config.js` (f.e. to enable nesting)

Reply:
> Seem to be addressed in: #5908

Came up with a possible fix for the issue but there's an issue. Applying changes to the `postcss.config.js` file only has effect after restarting the dev server.
I just wanted to know if it was an issue before, so that it will get addressed in another PR and doesn't block the current one.

Reply:
I think it was looking at the old code. I'd be fine merging it first, then handle the restart later.

Reply:
> I think it was looking at the old code. I'd be fine merging it first, then handle the restart later.

Sounds good, thanks!